Lib/test/test_gdb.py

```diff
@@ -247,6 +243,9 @@ def get_stack_trace(self, source=None, script=None,
         for pattern in (
             '(frame information optimized out)',
             'Unable to read information on python frame',
+            # gh-91960: On Python built with "clang -Og", gdb gets
+            # "frame=<optimized out>" for _PyEval_EvalFrameDefault() parameter
+            '(unable to read python frame information)',
```

Comment:
> Any idea why libpython.py prints this rather than "Unable to read information on python frame"? It's possible that there are just some spots that you missed back in https://github.com/python/cpython/pull/19081

It's likely possible to unify python-gdb.py logs, or use a different regex to match both cases :-) The ``(unable to read ...)`` format is for ``py-bt`` command. Depending on the command, the log is displayed differently. I took the lazy way of just matching the two strings.